setup.cfg

```diff
@@ -56,34 +56,10 @@ s3 =
     botocore==1.29.50
 mongodb =
     motor==3.1.1
-dev =
-    pytest==7.2.1
-    pytest-asyncio==0.20.3
-    pytest-cov==4.0.0
-    mypy==1.0.0
-    pylint==2.15.10
-    click==8.1.3
-    black==22.12.0
-    flake8==6.0.0
-    isort==5.11.5
-    bandit==1.7.4
-    pre-commit==2.21.0
-    requests==2.28.1
-    types-requests==2.28.11.7
-    mkdocs==1.4.2
-    mkdocs-autorefs==0.4.1
-    mkdocs-material==9.0.3
-    mkdocs-material-extensions==1.1.1
-    mkdocstrings==0.19.1
-    mkdocstrings-python-legacy==0.2.3
-    # 3.4.1 -> 3.4.2 causes issues in hexkit, but 3.7.1 works again
-    # 3.7.1 currently does not work with ghga-connector and dcs
-    testcontainers[kafka,mongo]==3.4.1
```

Comment:
Unfortunately, we have to keep at least some of the dependencies in here because they are used by other microservices when using the test hexkit test utils. Maybe we can split this up into: akafka-test, s3-test, and mongodb-test